src/views/SessionView/SessionHeader.vue

```diff
@@ -187,6 +188,7 @@ export default {
         SessionService.endSession(this, sessionId)
           .then(() => {
             this.$socket.disconnect();
+            this.$store.dispatch("user/sessionDisconnected");
```

Comment:
i think theres a chance to dry up some code here, you can move the if statement to inside the `then`

```
SessionService.endSession(this, sessionId)
  .then(() => {
    this.$socket.disconnect();
    this.$store.dispatch("user/sessionDisconnected");
    let url = "/";
    if (volunteerId) {
      url =
              "/feedback/" +
              sessionId +
              "/" +
              topic +
              "/" +
              subTopic +
              "/" +
              (this.user.isVolunteer ? "volunteer" : "student") +
              "/" +
              studentId +
              "/" +
              volunteerId;
    }
    router.push(url);
  })
  .catch(this.alertCouldNotEnd);
```